chain/chunks/src/metrics.rs

```diff
@@ -19,3 +19,12 @@ pub static PARTIAL_ENCODED_CHUNK_REQUEST_PROCESSING_TIME: Lazy<near_metrics::His
         )
         .unwrap()
     });
+
+pub static DISTRIBUTE_ENCODED_CHUNK_TIME: Lazy<near_metrics::Histogram> = Lazy::new(|| {
```

Comment:
I'd mention in the comment here, that this covers the preparation of network messages and passing it to peer manager. (but doesn't cover any time spent in peer_manager)